numpy/polynomial/polynomial.py

```diff
@@ -662,6 +662,10 @@ def polyval(x, c, tensor=True):
     Trailing zeros in the coefficients will be used in the evaluation, so
     they should be avoided if efficiency is a concern.
 
+    Do not confuse this function with the very similar
```

Comment:
In a similar vein to my comment above, maybe something like "Note that this function is preferred over the very similar top-level `numpy.polyval`..."